autoscheduler/frontend/src/types.ts

```diff
@@ -1,3 +1,48 @@
+/* eslint-disable import/prefer-default-export */
+
 /**
  * This file is where we will declare the types of each data-holding class we will be using.
  */
+export enum MeetingType {
+  LEC, LAB, RES, INS
+}
+
+export interface Meeting {
+  id: number;
+  crn: number;
+  building: string | null;
+  meetingDays: boolean[];
+  startTimeHours: number;
+  startTimeMinutes: number;
+  endTimeHours: number;
+  endTimeMinutes: number;
```

Comment:
Is there any benefit to storing these as separate values rather than two `date` objects(or rather, only the time within the date object)?

Reply:
The JS `Date` object will have some extraneous data for the year, month, date, seconds, and milliseconds, all of which will be filled with BS data. The space differences for these approaches are negligible, so whichever syntax produces the cleaner syntax should be preferred. I chose to prioritize clearly defining which data we are using over minimizing the number of fields. If this leads to messy syntax in your opinion, we can change it.